dev-python/flask-migrate/flask-migrate-2.5.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+inherit distutils-r1
+
+MY_PN="Flask-Migrate"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="SQLAlchemy database migrations for Flask applications using Alembic"
+HOMEPAGE="https://pypi.org/project/Flask-Migrate/"
+SRC_URI="mirror://pypi/${MY_P:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-python/alembic-0.7[${PYTHON_USEDEP}]
+	>=dev-python/flask-0.9[${PYTHON_USEDEP}]
+	>=dev-python/flask-sqlalchemy-1.0[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
```

Comment:
Since you're touching them, could you:

1. Remove explicit deps on setuptools (redundant).
2. Remove redundant DEPEND=$RDEPEND (this is needed only for tests most of the time, if you use d_e_t i will make that dep for you).
3. DEPEND → BDEPEND.